src/app/components/Transcript/README.md

```diff
@@ -0,0 +1,17 @@
+## Description
+
+The `Transcript` component renders video transcripts.
+
+## Props
+
+| Name            | type    | Description                                           |
+| --------------- | ------- | ----------------------------------------------------- |
+| transcript      | object  | contains transcript content                           |
+| title           | string  | title of video                                        |
+| hideDisclaimer? | boolean | decides whether to show disclaimer (defaults to true) |
```

Comment:
```suggestion
| hideDisclaimer | boolean | decides whether to show disclaimer (defaults to true) |
```

Reply:
I wonder if this is a confusing prop name if it's always `true`? Would it be better to name it `showDisclaimer` and default it to `false`? That way we are intentional when setting it to true i.e. we have to explicitly pass it in as a prop?